### ISSUE TITLE
Inconsitency in file license-list-data/json/licenses.json

### ISSUE THREAD

**Issue by vargenau** [spdx/license-list-XML#1676]
In file ```license-list-data/json/licenses.json``` you have:

```
    {
      "reference": "https://spdx.org/licenses/GPL-3.0.html",
      "isDeprecatedLicenseId": true,
      "detailsUrl": "https://spdx.org/licenses/GPL-3.0.json",
      "referenceNumber": 242,
      "name": "GNU General Public License v3.0 only",
      "licenseId": "GPL-3.0",
      "seeAlso": [
        "https://www.gnu.org/licenses/gpl-3.0-standalone.html",
        "https://opensource.org/licenses/GPL-3.0"
      ],
      "isOsiApproved": true,
      "isFsfLibre": true
    },

```

```
    {
      "reference": "https://spdx.org/licenses/GPL-3.0-only.html",
      "isDeprecatedLicenseId": false,
      "detailsUrl": "https://spdx.org/licenses/GPL-3.0-only.json",
      "referenceNumber": 274,
      "name": "GNU General Public License v3.0 only",
      "licenseId": "GPL-3.0-only",
      "seeAlso": [
        "https://www.gnu.org/licenses/gpl-3.0-standalone.html",
        "https://opensource.org/licenses/GPL-3.0"
      ],
      "isOsiApproved": true,
      "isFsfLibre": true
    },
```

In short, you have the same "name" for both the deprecated license and the replacement license.

This is true for:
```
             "name": "GNU General Public License v1.0 only",
             "name": "GNU General Public License v1.0 or later",
             "name": "GNU General Public License v2.0 only",
             "name": "GNU General Public License v2.0 or later",
             "name": "GNU General Public License v3.0 only",
             "name": "GNU General Public License v3.0 or later",
             "name": "GNU Lesser General Public License v2.1 only",
             "name": "GNU Lesser General Public License v3.0 only",
             "name": "GNU Lesser General Public License v3.0 or later",
             "name": "GNU Library General Public License v2 only",
             "name": "GNU Library General Public License v2 or later",
             "name": "Standard ML of New Jersey License",
```
But it is not the case for other deprecated licenses.

For example, you have:
```
   {
      "reference": "https://spdx.org/licenses/LGPL-2.1-or-later.html",
      "isDeprecatedLicenseId": false,
      "detailsUrl": "https://spdx.org/licenses/LGPL-2.1-or-later.json",
      "referenceNumber": 31,
      "name": "GNU Lesser General Public License v2.1 or later",
      "licenseId": "LGPL-2.1-or-later",
      "seeAlso": [
        "https://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html",
        "https://opensource.org/licenses/LGPL-2.1"
      ],
      "isOsiApproved": true,
      "isFsfLibre": true
    },
```
and
```
    {
      "reference": "https://spdx.org/licenses/LGPL-2.1+.html",
      "isDeprecatedLicenseId": true,
      "detailsUrl": "https://spdx.org/licenses/LGPL-2.1+.json",
      "referenceNumber": 320,
      "name": "GNU Library General Public License v2.1 or later",
      "licenseId": "LGPL-2.1+",
      "seeAlso": [
        "https://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html",
        "https://opensource.org/licenses/LGPL-2.1"
      ],
      "isOsiApproved": true,
      "isFsfLibre": true
    },
```

Also:
```
   {
      "reference": "https://spdx.org/licenses/AGPL-3.0-only.html",
      "isDeprecatedLicenseId": false,
      "detailsUrl": "https://spdx.org/licenses/AGPL-3.0-only.json",
      "referenceNumber": 343,
      "name": "GNU Affero General Public License v3.0 only",
      "licenseId": "AGPL-3.0-only",
      "seeAlso": [
        "https://www.gnu.org/licenses/agpl.txt",
        "https://opensource.org/licenses/AGPL-3.0"
      ],
      "isOsiApproved": true,
      "isFsfLibre": true
    },
```
and
```
    {
      "reference": "https://spdx.org/licenses/AGPL-3.0.html",
      "isDeprecatedLicenseId": true,
      "detailsUrl": "https://spdx.org/licenses/AGPL-3.0.json",
      "referenceNumber": 70,
      "name": "GNU Affero General Public License v3.0",
      "licenseId": "AGPL-3.0",
      "seeAlso": [
        "https://www.gnu.org/licenses/agpl.txt",
        "https://opensource.org/licenses/AGPL-3.0"
      ],
      "isOsiApproved": true,
      "isFsfLibre": true
    },
```

I would expect that you have the same "name" for both the deprecated license and the replacement license.

**Comment by vargenau** [comment]
Hi @goneall 

Could you please have a look at this issue?

**Comment by goneall** [comment]
@vargenau The name is getting pulled from the license XML `name` attribute on the `license` element.

This is more of a policy question as to whether names should be the same or different and if there even needs to be a consistency on the names for deprecated license names.

Once the policy is decided, the license XML's may need to be updated.  Since the names are more informative and are not required to be unique, I think it would be OK to update the names for existing licenses.

**Comment by vargenau** [comment]
Hi @goneall

I have created pull requests: #1710 #1711 #1712 #1713 #1714 #1715 #1716

**Comment by goneall** [comment]
Thanks @vargenau - the license-list-data should update within an hour of those PR's being merged.

**Comment by vargenau** [comment]
Hi @goneall

Could you please merge these pull requests?

Thank you

**Comment by goneall** [comment]
@jlovejoy @swinslow @zvr - Could I get one more review on the above referenced PR's - or let me know if you'd like me to just merge.

**Comment by swinslow** [comment]
Thanks @vargenau for the attention to detail on these!

* I've merged the fixes for the typos in #1710 and #1711.
* For the GFDL edits (#1714, #1715 and #1716), based on @jlovejoy's comment at https://github.com/spdx/license-list-XML/pull/1714#issuecomment-1362375350 I've closed those PRs.
* For the AGPL-1.0 one at #1712 and AGPL-3.0 at #1713, I'm going to close those also. Similar to the GFDL licenses, those did not have versions originally on the license list with `+` embedded in the license ID, so for the same reasons that @jlovejoy described in her comment, I don't think we would want to retroactively introduce an "only" presumption for those now-deprecated IDs.

Based on that, I'm going to go ahead and close this issue after closing #1712 and #1713. Thanks all!
